src/containers/Admin/LogoTab/CustomURL/index.tsx

```diff
@@ -0,0 +1,142 @@
+import React, { useState } from 'react'
+
+import {
+    Label,
+    Heading3,
+    Paragraph,
+    EmphasizedText,
+    StrongText,
+} from '@entur/typography'
+import { TextField } from '@entur/form'
+import { SecondarySquareButton } from '@entur/button'
+import { CheckIcon } from '@entur/icons'
+import { Tooltip } from '@entur/tooltip'
+
+import {
+    copySettingsToNewId,
+    setIdToBeDeleted,
+} from '../../../../services/firebase'
+
+import { useSettingsContext } from '../../../../settings'
+import { getDocumentId } from '../../../../utils'
+
+import '../styles.scss'
+
+enum inputFeedback {
+    ID_UNAVAILABE = 'Ikke ledig: Denne Tavla-linken er dessverre i bruk.',
```

Comment:
Bruk helst ordet "lenke" i staden for "link".

Reply:
ID_UNAVAILABE -> ID_UNAVAILABLE

Reply:
Fikset!